.python-version

```
@@ -0,0 +1 @@
+3.5.3
```

I don't know, but since it's a fork of the ruby equivalent, I'm guessing that it will error out and tell them they need to install 3.5.3. That's the desired behavior, IMO. I.e., fail fast and clearly. That's the best user experience.